data/json/mutations/mutations.json

```diff
@@ -5273,6 +5272,7 @@
     "description": "You can sense things you never imagined.  +7 Perception.",
     "types": [ "PER" ],
     "prereqs": [ "PER_UP_3" ],
+    "leads_to": [ "BIRD_EYE" ],
```

Comment:
Shouldn’t this be changes too? Otherwise it will be Perception +11 total instead of 7

Reply:
No I get that you can mutate them both currently in game but that doesn't exactly make it intended.  What would make it intended if this is the most significant ability that bird gets far and away.  I'm inclined to believe that's likely true given that bird mutants don't regenerate or fly or any other big post thresh abilities that I saw looking around.  I would guess that Bird is probably in an interim state while waiting for a significant retooling because +11 Perception isn't a playstyle changing group of mutations.

Reply:
yeah it's pretty clearly meant for Bird to get both for +11 perc total. Perc is the weakest stat anyways so it's not exactly harmful. Someone just made it Perceptive only and not Very/Extremely/Insanely.

Reply:
That's fair, I happen to think that puts this in line for a rework to in some way change the game loop.  "See's good" isn't that useful unless it greatly increases chance to hit the best weakpoints and even then it would be pretty niche unless you are playing Dark Days of the Dead, headshots only.